packages/cppo/cppo.1.7.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "Code preprocessor like cpp for OCaml"
+description: """\
+Cppo is an equivalent of the C preprocessor for OCaml programs.
+It allows the definition of simple macros and file inclusion.
+
+Cppo is:
+
+* more OCaml-friendly than cpp
+* easy to learn without consulting a manual
+* reasonably fast
+* simple to install and to maintain"""
+maintainer: [
+  "Martin Jambon <martin@mjambon.com>" "Yishuai Li <yishuai@upenn.edu>"
+]
+authors: "Martin Jambon"
+license: "BSD-3-Clause"
+homepage: "https://github.com/ocaml-community/cppo"
+doc: "https://ocaml-community.github.io/cppo"
+bug-reports: "https://github.com/ocaml-community/cppo/issues"
+depends: [
+  "ocaml" {>= "4.02.3"}
+  "dune" {>= "2.0"}
+  "base-unix"
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```

Reply:
Thanks for spotting these problems. Apparently a side effect of the fact that `cppo` was pinned (on my machine) when I ran `opam publish`.

Reply:
You're welcome. The `pinned` was an earlier flag used here -- but since some time, `dev` is the preferred flag.